tests/testthat/test-failed-chains.R

```diff
@@ -201,10 +200,15 @@ test_that("gq chains error on wrong input CSV", {
       "Mismatch between model and fitted_parameters csv"
     )
   )
+  if (cmdstan_version() < "2.26") {
+    err_msg <- "Error reading fitted param names"
+  } else {
+    err_msg <- "Mismatch between model and fitted_parameters csv file"
+  }
```

Comment:
GQ accepts CSV's now even if lp__ isnt the first column hence the different error.